### PR TITLE
Getting Ready for release 3.0.1-BETA

### PR DESCRIPTION
**Pull request by kryalama**
update changelog for release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
 # CHANGELOG
 
-# Version 3.0.1 GA
-* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp
+# Version 3.0.1-BETA
+* Update to otel-java-instrumentation 0.12.0
```
Comment (heyams) [inline CHANGELOG.md:4]: we usually don't mention this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+* Friendly error messages thrown for the following scenarios
+    - Missing connection string
+    - Invalid SSL certificate issues when not able to connect to IngestionEndPoint Url, Live endpoint Url and CdsProfiler endpoint url.
+    - Invalid Telemetry Processor Configuration
+* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp.
+* Map service.version to application_Version(Resolves [#1392](https://github.com/microsoft/ApplicationInsights-Java/issues/1392))
+* Lazy set propagators in Azure Functions
```
Comment (heyams) [inline CHANGELOG.md:11]: this is for az function only.  we don't need to mention it.  it was part of GA actually.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+* Bring in fix for Azure Functions instrumentation
```
Comment (heyams) [inline CHANGELOG.md:12]: we don't need to mention this either.. @trask can confirm this.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+* Fix NullPointerException for Linux consumption plan
```
Comment (heyams) [inline CHANGELOG.md:13]: no need to mention this.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,17 @@
...
-* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp
+# Version 3.0.1-BETA
+* Friendly error messages thrown for the following scenarios
+    - Missing connection string
+    - Invalid SSL certificate issues when not able to connect to IngestionEndPoint Url, Live endpoint Url and CdsProfiler endpoint url.
+    - Invalid Telemetry Processor Configuration
+* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp.
+* Map service.version to application_Version(Resolves [#1392](https://github.com/microsoft/ApplicationInsights-Java/issues/1392))
+* This release also brings interoperability with 2.x SDK:
+    - ThreadContext.getRequestTelemetryContext().getHttpRequestTelemetry().getProperties().put("key1", "val1")
```
Comment (heyams) [inline CHANGELOG.md:11]: we can delete line 11 & 12. doc will have the details.  might be helpful to paste the link to the doc here.

Comment (heyams) [reply CHANGELOG.md:11]: @trask can you review this?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,7 +1,17 @@
...
+	- ThreadContext.getRequestTelemetryContext().getHttpRequestTelemetry().setName("new name")
```
Comment (trask) [inline CHANGELOG.md:12]:
```suggestion
	- ThreadContext.getRequestTelemetryContext().getHttpRequestTelemetry().setName("new name")
	- ThreadContext.getRequestTelemetryContext().getHttpRequestTelemetry().getContext().getUser().setId("uname")

```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,17 @@
...
-# Version 3.0.1 GA
-* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp
+# Version 3.0.1-BETA
+* Friendly error messages thrown for the following scenarios
+    - Missing connection string
+    - Invalid SSL certificate issues when not able to connect to IngestionEndPoint Url, Live endpoint Url and CdsProfiler endpoint url.
+    - Invalid Telemetry Processor Configuration
+* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp.
+* Map service.version to application_Version(Resolves [#1392](https://github.com/microsoft/ApplicationInsights-Java/issues/1392))
+* This release also brings interoperability with 2.x SDK:
```
Comment (trask) [inline CHANGELOG.md:10]:
```suggestion
* This release also brings more interoperability with the 2.x SDK:
```

**Review by trask — APPROVED** [review]

**Review by heyams — APPROVED** [review]

**Pull request merged**